nanoFirmwareFlasher.Tests/FirmwarePackage_Tests.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why rename and add underscore? Code style call for camel case for class names.

Reply:
Ok